spec/models/chargeback_rate_detail_currency_spec.rb

```diff
@@ -1,31 +1,30 @@
 describe ChargebackRateDetailCurrency do
   describe '.seed' do
-    before do
+    it "returns supported currencies" do
       ChargebackRateDetailCurrency.seed
-    end
 
-    let(:expected_currencies) do
-      %w(AED AFN ALL AMD ANG AOA ARS AUD AWG AZN BAM BBD BDT BGN BHD BIF BMD BND BOB BRL BSD BTN BWP BYN BYR BZD CAD CDF CHF CLF CLP CNY COP CRC CUC CUP CVE CZK DJF DKK DOP DZD EGP ERN ETB EUR FJD FKP GBP GEL GHS GIP GMD GNF GTQ GYD HKD HNL HRK HTG HUF IDR ILS INR IQD IRR ISK JMD JOD JPY KES KGS KHR KMF KPW KRW KWD KYD KZT LAK LBP LKR LRD LSL LYD MAD MDL MGA MKD MMK MNT MOP MRO MUR MVR MWK MXN MYR MZN NAD NGN NIO NOK NPR NZD OMR PAB PEN PGK PHP PKR PLN PYG QAR RON RSD RUB RWF SAR SBD SCR SDG SEK SGD SHP SKK SLL SOS SRD SSP STD SVC SYP SZL THB TJS TMT TND TOP TRY TTD TWD TZS UAH UGX USD UYU VES VND VUV WST XAF XAG XAU XCD XDR XOF XPD XPF XPT YER ZAR ZMK ZMW)
-    end
+      expected_currencies = %w[AED AFN ALL AMD ANG AOA ARS AUD AWG AZN BAM BBD BDT BGN BHD BIF BMD BND BOB BRL BSD BTN BWP BYN BYR BZD CAD CDF CHF CLF CLP CNY COP CRC CUC CUP CVE CZK DJF DKK DOP DZD EGP ERN ETB EUR FJD FKP GBP GEL GHS GIP GMD GNF GTQ GYD HKD HNL HRK HTG HUF IDR ILS INR IQD IRR ISK JMD JOD JPY KES KGS KHR KMF KPW KRW KWD KYD KZT LAK LBP LKR LRD LSL LYD MAD MDL MGA MKD MMK MNT MOP MRU MUR MVR MWK MXN MYR MZN NAD NGN NIO NOK NPR NZD OMR PAB PEN PGK PHP PKR PLN PYG QAR RON RSD RUB RWF SAR SBD SCR SDG SEK SGD SHP SKK SLL SOS SRD SSP STD SVC SYP SZL THB TJS TMT TND TOP TRY TTD TWD TZS UAH UGX USD UYU UZS VES VND VUV WST XAF XAG XAU XCD XDR XOF XPD XPF XPT YER ZAR ZMK ZMW]
```

Comment:
Hard to tell in the PR diff (easier to see in the first commit's diff), but there are 2 changes here...

- Added UZS
- Renamed MRO to MRU

The latter is interesting because it seems seed doesn't delete old currencies, nor does it have a way to "update" old currencies where the code has changed.  I'm not sure if that's a good thing or a bad thing. 

@lpilcher Thoughts?  Should we have a data migration moving MRO to MRU, and then delete old currencies?  Or should we just leave it the way it is, which is technically backwards compatibile?

Reply:
I'm fine with keeping this for now but this test feels brittle.  We'll have to change this whenever money adds/removes/renames currencies, right?  Maybe we should be looking for specific subset of newer currencies we want to test against?

I do like the dropping of the currency count test though.  👍

Reply:
Yeah, the test is brittle, and a subset is the only thing I can think of as well.  I think that's better as a follow up PR though.

Reply:
yeah, let's do the followup after this PR is merged.

Reply:
> @lpichler Thoughts? Should we have a data migration moving MRO to MRU, and then delete old currencies? Or should we just leave it the way it is, which is technically backwards compatibile?

@Fryguy Change MRO to MRU is basically: removing one old currency and adding new.
There are different they have different [rate.](https://en.wikipedia.org/wiki/Mauritanian_ouguiya)

So if anybody used MRO - I prefer to keep MRO as he selected it.

If anybody created report/chargeback rate/services/,,, with MRO and we will change it to MRU without conversions of prices and are changing meaning of it - so I suggest don't change it.

If there will be change in symbol because of typo,.. we can  create migration symbol is unique identification for seeding.

> We'll have to change this whenever money adds/removes/renames currencies, right?

this is a reason why there is the test - to be informed about external changes.


anyway do you think that we can copy their json with currencies to our project ? (I don't know how to judge this) because we need (or we will soon) money gem just because of this file(list of currencies).

thanks!